controllers/imagecollector/imagecollector_controller.go

```diff
@@ -163,13 +170,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 				},
 			},
 		}
-		log.Info("Queued first ImageCollector reconcile")
-	}()
+
+		log.Info("...Queued first ImageCollector reconcile")
+	})
 
 	return nil
 }
 
 //+kubebuilder:rbac:groups="batch",resources=jobs,verbs=get;list;create;delete;watch
```

Comment:
can we remove this?

Reply:
We can remove

Reply:
i still see this, do we want to remove in this pr?

Reply:
For clarity, you meant the kubebuilder comment? Or the log.Info?

Reply:
sorry, should have clarified. I meant kubebuilder marker